src/Grpc.Net.Client.Web/Internal/GrpcWebRequestContent.cs

```diff
@@ -56,6 +56,12 @@ protected override async Task SerializeToStreamAsync(Stream stream, TransportCon
                 }
 
                 await _inner.CopyToAsync(stream).ConfigureAwait(false);
+
+                if (base64RequestStream != null)
+                {
+                    // Flush any remaining base64 content
+                    await base64RequestStream.FlushAsync().ConfigureAwait(false);
+                }
```

Comment:
I don't know how often _mode is something other than GrpcWebText, but consider changing the implementation to something like:
```C#
protected override Task SerializeToStreamAsync(Stream stream, TransportContext context) =>
    _mode == GrpcWebMode.GrpcWebText ? SerializeTextToStreamAsync(stream) :
    _inner.CopyToAsync(stream);

private async Task SerializeTextToStreamAsync(Stream stream)
{
    using var base64 = new Base64RequestStream(stream);
    await _inner.CopyToAsync(base64).ConfigureAwait(false);
    await base64.FlushAsync().ConfigureAwait(false);
}
```